src/Command/AbstractCommand.php

```diff
@@ -106,4 +106,19 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
 
         return str_replace('.git', '', end($repositoryArray));
     }
+
+    /**
+     * Return the last packagist version.
+     */
+    final protected function getLastPackageVersion(Package $package): Package\Version
+    {
+        $versions = $package->getVersions();
+        $lastVersion = reset($versions);
+
+        if (false === $lastVersion) {
+            throw new \LogicException('The package has no version');
```

Comment:
In this case we cannot update the settings of a new repo because this exception is not caught... 🤔

Reply:
Fixed, I use an empty version now.